sigs.yaml

```
@@ -1204,13 +1204,21 @@ sigs:
     - https://raw.githubusercontent.com/kubernetes-sigs/contributor-playground/master/OWNERS
     - https://raw.githubusercontent.com/kubernetes/community/master/mentoring/OWNERS
     meetings:
-    - description: sig-contribex mentoring subproject meeting
-      day: Friday
-      time: "10:00"
+    - description: sig-contribex mentoring subproject meeting (APAC Friendly Time)
+      day: Monday
+      time: "20:30"
       tz: PT
       frequency: biweekly
```

```suggestion
      frequency: 2nd and 4th Tuesday of each month
```
Technically it's not biweekly. It's the 2nd and 4th tuesday of the month. Was kind of a pain to schedule that way because it was 2  separate events per meeting and having to wait to schedule each out to get around calendar spam queue lol